### PR TITLE
strickter loglevel syntax verification

### PR DESCRIPTION
**Pull request by igt-marcin-wasilewski**
Apache::LogLevel allows invalid strings like 'warning' which are not accepted by apache httpd

### CODE REVIEW

File: types/loglevel.pp [hunk 1]
```diff
@@ -24,4 +24,4 @@
 # * `trace8`
 #
 # @see https://httpd.apache.org/docs/current/mod/core.html#loglevel
-type Apache::LogLevel = Pattern[/(emerg|alert|crit|error|warn|notice|info|debug|trace[1-8])/]
+type Apache::LogLevel = Pattern[/\A(([a-z_\.]+:)?(emerg|alert|crit|error|warn|notice|info|debug|trace[1-8])\W*)+\Z/]
```
Comment (smortex) [inline types/loglevel.pp:27]: This match `errorerror` which is not valid.

Matching `λ(\s+λ)*` (where λ match a log-level) should help:

```suggestion
type Apache::LogLevel = Pattern[/\A([a-z_\.]+:)?(emerg|alert|crit|error|warn|notice|info|debug|trace[1-8])(\s+([a-z_\.]+:)?(emerg|alert|crit|error|warn|notice|info|debug|trace[1-8]))*\Z/]
```

**Review by smortex — APPROVED** [review]
LGTM!

**Comment by smortex** [comment]
> Very sorry for the long delay in approval for this PR. We have been busy with key maintenance around our modules.

@LukasAud, has the situation improved? :grin:

**Comment by Ramesh7** [comment]
Thanks @smortex for your patience, sorry for delay as this came to my radar just now.

**Pull request merged**